shared/common-adapters/relative-popup-hoc.desktop.js

```diff
@@ -245,11 +245,21 @@ function ModalPositionRelative<PP>(
     }
 
     render() {
+      // React will complain if WrappedComponent is a HTMLElement and we try to attach these props
+      const noPassProps = ['targetRect', 'position', 'onClosePopup']
+      // $ForceType thinks {} is invalid for PP, we're filtering out the HOC's props
```

Comment:
you can likely fix this by doing the noPassProps check in reduce and not set res[k] in that instance